packages/sdk/wallets/wallet-walletconnect/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@celo/wallet-walletconnect",
+  "version": "1.0.3-dev",
+  "description": "WalletConnect wallet implementation",
+  "author": "Celo",
+  "license": "Apache-2.0",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "keywords": [
+    "celo",
+    "blockchain",
+    "sdk",
+    "WalletConnect",
+    "wallet"
+  ],
+  "scripts": {
+    "build": "tsc -b .",
+    "clean": "tsc -b . --clean",
+    "docs": "typedoc && ts-node ../../utils/scripts/linkdocs.ts wallet-walletconnect",
+    "test": "jest --runInBand",
+    "lint": "tslint -c tslint.json --project .",
+    "prepublishOnly": "yarn build"
+  },
+  "dependencies": {
+    "@celo/connect": "1.0.3-dev",
+    "@celo/utils": "1.0.3-dev",
+    "@celo/wallet-base": "1.0.3-dev",
+    "@celo/wallet-remote": "1.0.3-dev",
+    "@walletconnect/client": "2.0.0-alpha.26",
+    "bignumber.js": "^9.0.0",
+    "debug": "^4.1.1"
+  },
+  "devDependencies": {
+    "@celo/contractkit": "1.0.3-dev",
+    "@celo/dev-utils": "0.0.1-dev"
+  },
+  "engines": {
+    "node": ">=8.13.0"
```

Comment:
>= 10? Or just = 10?

Reply:
I know the other ones say node <= 10, but my cli also isn't happy if I'm on anything but 10. This might be something we have to update everywhere else, though.